textproto/header.go

```diff
@@ -500,11 +507,12 @@ func ReadHeader(r *bufio.Reader) (Header, error) {
 			return newHeader(fs), err
 		}
 
-		return newHeader(fs), fmt.Errorf("message: malformed MIME header initial line: %v", string(line))
+		return newHeader(fs), fmt.Errorf("%w: %v", ErrMalformedMimeHeaderInittialLine, string(line))
```

Comment:
Why it is done using Errorf instead of creating an error type?

Reply:
Because it attaches the line as string to the error, but can be still checked with errors.Is() ErrMalformedMimeHeaderInittialLine. Please see https://blog.golang.org/go1.13-errors 

With a static error type this is more complicated.